docs/api/javascript-api.md

```diff
@@ -61,7 +61,8 @@ The best source of docs is the main [ReactOnRails.js](https://github.com/shakaco
   /**
    * Set options for ReactOnRails, typically before you call ReactOnRails.register
    * Available Options:
-   * `traceTurbolinks: true|false Gives you debugging messages on Turbolinks events
+   * `traceTurbolinks: true|false` Gives you debugging messages on Turbolinks events
+   * `turbo: true|false` enables Turbo, which is not auto-detected like the Turbolinks library is
```

Comment:
@Judahmeek do we need to update this here as it doesn't seem to include `turbo`?

<img width="305" alt="Screen Shot 2022-01-14 at 3 39 50 PM" src="https://user-images.githubusercontent.com/3027970/149598613-18b23f01-5925-46e7-8105-dac3ac76acf8.png">

Reply:
we could allow for either string, but it's not totally required.

Reply:
Yeah I thinking `turbo?: boolean`

Or `turbo: boolean | null` or however it's actually defined. 

I think it'd help increase visibility because my IDE only shows `traceTurboLinks`.

Reply:
if we could easily just have the shorter traceTurbo, that's better for the long-term.

Reply:
@Judahmeek, yes, we need to update the d.ts file.

Reply:
@justin808 my interpretation of the code seems slightly different.

For Turbo to work I need:

`turbo: true`

To see the logging I need:

`traceTurboLinks: true`

Both have to be on for it to work and log with Turbo. 

--

Turbolinks seems to work without opting into it. `traceTurboLinks: true` just prints to the logs in that scenario.